src/Native/Unix/System.Native/pal_networking.c

```diff
@@ -1749,7 +1750,9 @@ int32_t SystemNative_GetSockOpt(
                 value = value == 0 ? 1 : 0;
             }
             *(int32_t*)optionValue = value;
-
+#else
```

Comment:
Nit: Add comment `// !SO_REUSEPORT` for better readability and matching of ifdef-else